WheelOfFortune/Prompt.cs

```diff
@@ -151,7 +173,24 @@ public static string PromptInValidEmptyInput()
         public static void GameOverMessage(Player[] winners)
         {
 
+            if (winners.Length == 1)
+            {
+                Console.Write($"Player {winners[0].Name} won the Game.");
```

Comment:
I think you can use a WriteLine here since we're not prompting them for anything, and so that anything the follows start on a new line.